src/main/java/it/gov/pagopa/pu/debtpositions/service/InstallmentServiceImpl.java

```diff
@@ -73,4 +77,52 @@ public String updateInstallmentNotificationDate(UpdateInstallmentNotificationDat
     return debtPositionUpdateInstallmentService.updateInstallment(debtPositionDTO, updatedInstallments, wfExecutionParameters, accessToken, operatorExternalUserId).getRight();
   }
 
+  @Override
+  public InstallmentDTO updateInstallmentNotificationFee(Long organizationId, String nav, List<DebtPositionOrigin> debtPositionOrigin, long notificationFee) {
+    List<InstallmentDTO> installments = installmentPIIRepository.getByOrganizationIdAndNav(organizationId, nav, debtPositionOrigin).stream()
+      .filter(installment ->
+        installment.getStatus().equals(InstallmentStatus.UNPAID) ||
+        installment.getStatus().equals(InstallmentStatus.EXPIRED) ||
+        installment.getStatus().equals(InstallmentStatus.UNPAYABLE))
```

Comment:
pull from develop and use the Set it.gov.pagopa.pu.debtpositions.util.InstallmentUtils#MODIFIABLE_STATUSES instead